web/pom.xml

```diff
@@ -39,6 +39,12 @@
       <artifactId>json-simple</artifactId>
       <version>1.1</version>
     </dependency>
+    <!--  org.json  -->
```

Comment:
I would expect we already have sufficient json libraries included / supported already. Can you please check again whether this dependency is really needed?